connector-test.sh

```diff
@@ -10,7 +10,8 @@ sleep 10
 
 confluent local services connect status
 
-connectors="credit_cards stores transactions purchases"
+#connectors="credit_cards stores transactions purchases"
```

Comment:
you may add `campaign_finance` in the same list of connectors at the end.

Reply:
This can be changed depending on what is being tested.